app/components/loading_component/loading_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="w-full flex justify-center py-1">
+  <svg class="animate-spin -ml-1 mr-3 h-5 w-5 text-gray-500" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Comment:
Is this a heroicon? Let's drop it in `app/assets/images/icon/` and use `inline_svg_tag`.

Reply:
It's not a heroicon, but it is from Tailwind's website. I'll drop it in assets.

Reply:
You can put it in `assets/images/icons` and not nested under `outline/` or `solid/`.